src/app/workflow/workflow.component.ts

```diff
@@ -110,18 +115,45 @@ export class WorkflowComponent extends Entry {
     }
   }
 
-  private isWdl(workflowRef: ExtendedWorkflow) {
-    return workflowRef.full_workflow_path && workflowRef.descriptorType === 'wdl';
+  public isWdl(workflowRef: ExtendedWorkflow) {
```

Comment:
check how many times this is called with the same workflowRef

Reply:
approved if say...3 or less

Reply:
It was 6 times, but after a discussion went ahead with it and will address anew on develop branch.